### PR TITLE
Django 22

### PR DESCRIPTION
**Pull request by madhuBayy**
@ldko @somexpert 
This Django 2.2 upgrade is ready for review.

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-django==1.11.29
+django==2.2.17
```
Comment (somexpert) [inline requirements-test.txt]: I think it makes sense to use `~=` here too

File: tests/tests.py [hunk 2]
```diff
@@ -122,7 +121,6 @@ def test_date_and_levelConforms(self, input_date, levelConforms, edtf_output, cl
                                  ('15XX-12-25', '1', False),
                                  ('0000/0000', 1, False),
                                  ('1863- 03-29', '0', False),
-                                 (None, '0', False),
```
Comment (somexpert) [inline tests/tests.py:125]: What's the reason for these removals?

Comment (madhuBayy) [reply tests/tests.py:125]: In django 2.2, test client raises `TypeError` when using `None`. The docs suggest to replace `None` with an empty string.  If `date` is empty string it raises `HttpResponseBadRequest` and this case is already being tested [here](https://github.com/unt-libraries/django-edtf/blob/master/tests/tests.py#L61)

**Review by ldko — APPROVED** [review]
This works for me, though I concur about using the `~=` for the Django requirement.

**Review by somexpert — APPROVED** [review]
Looks good to me! :thumbsup:

**Pull request merged**